sync/cli/_databricks.py

```diff
@@ -173,6 +174,19 @@ def create_submission(
     return
 
 
+@click.command
+@click.argument("project", callback=validate_project)
+def create_recommendation(project: dict):
+    return create_project_recommendation(project["id"])
```

Comment:
What does it look like when you run this? It doesn't look like this command has any output.